### ISSUE TITLE
[Failing Test] verify.vendor failing in master-blocking:verify-master

### ISSUE THREAD

**Issue by smourapina** [kubernetes/kubernetes#77363]
**Which jobs are failing**:
`master-blocking:verify-master`

**Which test(s) are failing**:
[verify.vendor](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-verify-master/1123836606650781701)

**Since when has it been failing**:
Since 05/01.

**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-blocking#verify-master

**Reason for failure**:
```
Your vendored results are different:
diff -Naupr -x BUILD -x 'AUTHORS*' -x 'CONTRIBUTORS*' vendor/k8s.io/apimachinery/pkg/apis/meta/v1/generated.pb.go /home/prow/go/src/k8s.io/kubernetes/_tmp/kube-vendor.U1BxPD/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/generated.pb.go
--- vendor/k8s.io/apimachinery/pkg/apis/meta/v1/generated.pb.go	2019-05-02 06:59:15.941750743 +0000
+++ /home/prow/go/src/k8s.io/kubernetes/_tmp/kube-vendor.U1BxPD/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/generated.pb.go	2019-05-02 07:20:40.000000000 +0000
``

**Anything else we need to know**:
Started failing after commit [#6d691c998](https://github.com/kubernetes/kubernetes/compare/023066464...6d691c998?)

/milestone v1.15
/sig api-machinery
/kind failing-test
/priority important-soon

/cc @alejandrox1 @jimangel @rarchk @alenkacz

**Comment by neolit123** [comment]
/assign @liggit @smarterclayton 

as per:
https://github.com/kubernetes/kubernetes/pull/77136
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-verify-master/1123836606650781701

PTAL. thanks.

**Comment by alejandrox1** [comment]
Seem like #77136 also had something to do with bazel-test-master https://testgrid.k8s.io/sig-release-master-blocking#bazel-test-master :thinking:

**Comment by smarterclayton** [comment]
How did this merge and then later fail?

**Comment by liggitt** [comment]
looks like verify-protobuf doesn't pick up the meta packages?

```
git grep --files-with-matches -e '// +k8s:protobuf-gen=package' cmd pkg staging | xargs -n 1 dirname
```

**Comment by liggitt** [comment]
fixed in #77399
